build.sbt

```diff
@@ -519,18 +518,18 @@ lazy val scalap = configureAsSubproject(project)
   )
   .dependsOn(compiler)
 
-lazy val partestExtras = Project("partest-extras", file(".") / "src" / "partest-extras")
-  .dependsOn(replFrontend, scaladoc)
-  .settings(commonSettings)
-  .settings(generatePropertiesFileSettings)
-  .settings(clearSourceAndResourceDirectories)
+lazy val partest = configureAsSubproject(project)
+  .dependsOn(library, reflect, compiler, scalap, replFrontend, scaladoc)
   .settings(disableDocs)
-  .settings(disablePublishing)
   .settings(
-    name := "scala-partest-extras",
-    description := "Scala Compiler Testing Tool (compiler-specific extras)",
-    libraryDependencies += partestDep,
-    unmanagedSourceDirectories in Compile := List(baseDirectory.value)
+    name := "scala-partest",
```

Comment:
Is this intended to switch the group ID from `org.scala-lang.modules` to `org.scala-lang`? Otherwise an `organization` setting should be added.

You probably want to add the usual boilerplate from the other published projects, too:
```
  .settings(generatePropertiesFileSettings)
  .settings(Osgi.settings)
  .settings(AutomaticModuleName.settings("scala.partest"))
```

Reply:
Good question about the group ID. It seems more consistent to me to switch to `org.scala-lang`.

Thanks for the other suggestions.

Reply:
`generatePropertiesFileSettings` is part of `configureAsSubproject`, so that's redundant (no?)

Reply:
Other projects mention it, too, but it should be redundant. We can remove it everywhere.